packages/web-components/fast-foundation/CEMToMarkdown.mjs

```diff
@@ -216,7 +216,24 @@ function replaceJSDOCLinksWithMDLinks(text)
         {
             text = text.replace(jsdocLink, "[" + linkParts[1].trim() + "](" + linkParts[0].trim() + ")");
         }
-        startIndex = text.indexOf('{@link',endIndex);
+        startIndex = text.indexOf('{@link');
+    }
+    return text;
+}
+
+function removeCommentBlocks(text)
+{
+    let startIndex = text.indexOf('/*');
+    if(startIndex < 0) return text;
+
+    while(startIndex>=0)
+    {
+        let endIndex = text.indexOf("*/", startIndex);
+        if(endIndex < 0) return text;
+
+        const commentBlock = text.slice(startIndex, endIndex + 2);
+        text = text.replace(commentBlock, "");
+        startIndex = text.indexOf('/*');
     }
     return text;
 }
```

Comment:
This suggestion removes both block comments and inline comments, but it needs to be performed before newlines are replaced since inline comments are taken from `//` to the end of the line.

```suggestion
function removeCommentBlocks(text) {
    // Remove block comments
    text = text.replace(/\/\*(.|\s)*?\*\//gm, "");

    // Remove inline comments
    return text.replace(/\/\/.*$/gm, "");
}
```

Reply:
Done.